### PR TITLE
Use common build scripts in chroot env, too

### PR DESCRIPTION
**Pull request by TheAssassin**
Branch based on #67, therefore waiting for that one to be merged first.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# -*- mode: ruby -*-
```
Comment (probonopd) [inline Vagrantfile:1]: Even more complexity? I know that this probably makes your life easier but for many people this is something they don't understand. Is this now a third option for building, besides a simple chroot and Docker?

Comment (TheAssassin) [reply Vagrantfile:1]: This is an autogenerated template. Plus, using Vagrant is entirely optional. You don't have to use it. It's just to set up a VM that can be used for testing chroot builds. It's not a third option as you insinuate. Please read the commit message.

Comment (probonopd) [reply Vagrantfile:1]: Maybe this should be put more permanently visible in a comment inside the Vagrant file?

**Comment by TheAssassin** [comment]
I plan to write a short guide on vagrant in an upcoming PR.

**Review by probonopd — APPROVED** [review]

**Pull request merged**